addons/modules/functions/fnc_moduleThrowSmoke.sqf

```diff
@@ -0,0 +1,42 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Zeus module function to make unit throw a smoke grenade.
```

Comment:
Perhaps create a dialog with a dropdown for the colors of smoke the unit has? Rather than only white.

Reply:
The current modules were created with the focus of "here's what the players need to see". I'll be creating another module where Zeus can choose from the magazines available on the AI unit to cover this functionality.